src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/ContentEncryptionAlgorithmTests.cs

```diff
@@ -135,7 +182,7 @@ private static void VerifyAlgorithm3Des(byte[] encodedMessage)
         }
 
         [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
+        [ActiveIssue(10311, PlatformID.AnyUnix)]
```

Comment:
10311 is about an RC4 key length.  Why would that disable this test?

Thinking ahead: VerifyAlgorithmRc4 must check the KeyLength value.  Maybe it needs to be changed to
```c#
if (CanReadRc4KeyLength)
{
    Assert.Equals(the right number, expr);
}
else
{
    Assert.Equals(0, expr);
}
```